tools/autograd/gen_autograd.py

```diff
@@ -33,6 +33,9 @@
 # A map: function name => two options:
 #      1. name of the argument that all outputs are view of
 #      2. map: output idx => name of the argument that this result is view of
+# If you update the list below, you MUST also update the public list of view ops accordingly
```

Comment:
It would be the listS below. As the ones from RETURNS_VIEWS_OF_INPUT should be added as well.